### PR TITLE
US133575 updated end of unit screen for empty unit

### PR DESCRIPTION
**Pull request by ChrisKraljevicD2L**
## Relevant Rally Links

- https://rally1.rallydev.com/#/289692574792/custom/355050439968?detail=%2Fuserstory%2F614281188937

## Description

End of unit screen has a new design

## Goal/Solution

End of unit screen now reflects new designs

## Video/Screenshots

image

## Related PRs

- N/A

## Remaining Work

- [x] Dev Testing: another developer will test this code on their machine

### CODE REVIEW

**Comment by Kiefer-Co-D2L** [comment]
What's the way to view end of sequence for an empty unit?
image
I'm getting a 404 when I try to open an empty unit.

**Comment by ChrisKraljevicD2L** [comment]
> 

huh. I just made an empty unit in lessons, switched to viewing as a student, then opened the unit in LX

**Comment by ianlaidlaw** [comment]
it says there are conflicts in this pr but i don't see them in the file comparison. to be sure, i'll wait to review until they are resolved

File: components/d2l-sequences-content-eol-main.js [hunk 1]
```diff
@@ -265,6 +275,21 @@ export class D2LSequencesContentEoLMain extends D2L.Polymer.Mixins.Sequences.Ret
...
+			return false;
+		}
+
+		const summarySubEntity = entity.getSubEntityByRel('end-of-sequence-summary');
+
+		if (!summarySubEntity) {
+			return false;
+		}
+
+		debugger; //eslint-disable-line
```
Comment (ianlaidlaw) [inline components/d2l-sequences-content-eol-main.js:289]: not sure we want to keep this guy

Comment (ChrisKraljevicD2L) [reply components/d2l-sequences-content-eol-main.js:289]: good call

**Review by ianlaidlaw — APPROVED** [review]

**Comment by Kiefer-Co-D2L** [comment]
image
Dev test successful!

**Review by Kiefer-Co-D2L — APPROVED** [review]

File: components/d2l-sequences-content-eol-main.js [hunk 2]
```diff
@@ -30,39 +30,82 @@ export class D2LSequencesContentEoLMain extends D2L.Polymer.Mixins.Sequences.Ret
...
+							<d2l-sequences-content-eol-in-progress
+								href={{href}}
+								token="[[token]]"
+								showMissed=[[showMissed]]
+								missedCount={{missedCount}}>
+							</d2l-sequences-content-eol-in-progress>
+						</template>
+						<template is="dom-if" if="[[hasMissed]]">
+							<template is="dom-if" if="[[!showMissed]]">
+								<d2l-sequences-end-of-lesson-progress href="{{_getRootHref(entity)}}" token="[[token]]">
```
Comment (brissons) [inline components/d2l-sequences-content-eol-main.js:65]: Is this part in [d2l-sequences-content-eol-in-progress](https://github.com/BrightspaceHypermediaComponents/sequences/blob/0de2480bf6f8f58a9c01a724839ef5e50a71b40f/components/d2l-sequences-content-eol-in-progress.js)?

Comment (ChrisKraljevicD2L) [reply components/d2l-sequences-content-eol-main.js:65]: I believe you're correct. I'll fix that :+1:

Comment (ChrisKraljevicD2L) [reply components/d2l-sequences-content-eol-main.js]: yep that was some merge conflict leftovers I forgot. should be taken care of now.

**Review by brissons — APPROVED** [review]

**Pull request merged**